src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salaryFirst = 0;
+        int salarySecond = 0;
+        int salaryThird = 0;
+        String firstName = "";
+        String secondName = "";
+        String thirdName = "";
+
+        String [] splitDataFrom = dateFrom.split("[.]");
+        String [] splitDataTo = dateTo.split("[.]");
+        String [] splitCurrentData = new String[3];
+        String [] splitInputString = new String[4];
```

Comment:
```suggestion
        String[] splitDataTo = dateTo.split("[.]");
        String[] splitCurrentData = new String[3];
        String[] splitInputString = new String[4];
```
do you really need to initialize with  new String[3] and  new String[4] ?